bibpy/entry/entry.py

```diff
@@ -150,8 +150,15 @@ def extra_fields(self):
         return [ef for ef in self.fields if ef not in bibpy.fields.all]
 
     def get(self, name, default=None):
-        """Return the value for name or default if the name does not exist."""
-        return getattr(self, name, default)
+        """Return the value for name or default if the name does not exist.
+
+        This will also return the default if the name exists (is a valid field)
+        but is set to None.
+
+        """
+        value = getattr(self, name, None)
```

Comment:
this could just be `value = getattr(self, name, default)`?

Reply:
Hey @epignatelli. Thanks for looking at the PR. Your suggestion would work too yes.

We want to return the default whenever the field's value is `None` (unset from bibpy's perspective) or doesn't exist in the entry. Both the fix in this PR and your suggestion result in the desired behaviour.

I have created a [branch](https://github.com/MisanthropicBit/bibpy/tree/bump-to-v1.0.1) to bump to v1.0.1 with this fix and [another one](https://github.com/MisanthropicBit/bibpy/tree/move-to-github-actions) to move things from travis ci to github actions so the test suite etc. can run again.